doc/source/whatsnew/v0.24.0.txt

```diff
@@ -132,7 +132,7 @@ Strings
 Indexing
 ^^^^^^^^
 
--
+- Bug in :func:`pandas.core.arrays.categorical._factorize_from_iterable` inappropriately caused ``TypeError`` to be raised when an ``Enum`` was used as a factor in a ``MultiIndex`` (:issue:`21298`)
```

Comment:
Release notes shouldn't reference private methods.

You should be able to reference Enum with :class:`enum.Enum`, and :class:`MultiIndex`.